q/gradle-quickstart.sh

```diff
@@ -169,17 +168,23 @@ dependencies {
     // Compile-time dependencies that should NOT be part of the
     // shadow jar and are provided in the lib folder of Flink
     // --------------------------------------------------------------
-    compile "org.apache.flink:flink-java:\${flinkVersion}"
-    compile "org.apache.flink:flink-streaming-java_\${scalaBinaryVersion}:\${flinkVersion}"
+    if (flinkVersionNew) {
+        implementation "org.apache.flink:flink-streaming-java:\${flinkVersion}"
+        implementation "org.apache.flink:flink-clients:\${flinkVersion}"
+    } else {
+        implementation "org.apache.flink:flink-streaming-java_\${scalaBinaryVersion}:\${flinkVersion}"
+        implementation "org.apache.flink:flink-clients_\${scalaBinaryVersion}:${flinkVersion}"
```

Comment:
```suggestion
        implementation "org.apache.flink:flink-clients_\${scalaBinaryVersion}:\${flinkVersion}"
```

Reply:
Thx for catching that, but somehow, when testing the script it worked without the escape.
Anyways, fixed it, Thx!